content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```diff
@@ -71,15 +71,24 @@ config. Save it as `my-scheduler.yaml`:
 
 {{< codenew file="admin/sched/my-scheduler.yaml" >}}
 
-An important thing to note here is that the name of the scheduler specified as an
-argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the optional `spec.schedulerName` on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
+In the above manifest, you use a [Scheduler Configuration](/docs/reference/scheduling/config/)
```

Comment:
Is it better to refer to Scheduler Configuration as KubeSchedulerConfiguration?
If Scheduler Configuration is not an API object then it should be lower case

Reply:
@reylejano IMO, it is better to use `KubeSchedulerConfiguration` as this is a API resource type although it is not a server resource type.

@alculquicondor @damemi please correct me if I am wrong.

Reply:
@reylejano IMO, it is better to use `KubeSchedulerConfiguration` as this avoids any confusion regarding the resource type (this is an API resource type although it is not a server resource type). @alculquicondor @damemi WDYT?

Reply:
I usually try to use `KubeSchedulerConfiguration` to show that I am specifically talking about this file/API object (and not, for example, other types of config like flags/policy/etc...)

Reply:
`KubeSchedulerConfiguration` sgtm

Reply:
@damemi @alculquicondor ack. Will update.

Reply:
@reylejano updated in https://github.com/kubernetes/website/pull/30107/commits/7449bb36a0eaa7f2983fa8fae005e00f06496fa8.